### ISSUE TITLE
[BUG] Revision to answer FAQ Question: "Can I change the hostname, the IP address, protocol or port after the install?""

### ISSUE THREAD

**Issue by Aviel17** [pivpn/pivpn#908]
**Is your feature request related to a problem? Please describe.**
Yes; When I decided to change my Public DNS Address to the one I desired to use instead, I wasn't able to located nor edit the "Default.txt" text file in the directory stated in the answer (`/etc/openvpn/easy-rsa/pki/Default.txt`). *NOTE: this may be because I am trying to perform this operation on my Raspberry Pie 3.*

**Describe the solution you'd like**
As I rummaged through my `/etc/openvpn `folder, I found the `server.conf` file and was able to access it and change the appropriate settings detailed above.

**Describe alternatives you've considered**
N/A.

**Additional context**
For some reason, I wasn't able to edit the contents of the `server.conf` file without going into the terminal and sudo-ing the edits myself:

In other words, (from the home directory) just copy and paste this into the terminal:

`sudo nano /etc/openvpn/server.conf`

Then, just edit what you want!

Thank you all,
Aviel

**Comment by orazioedoardo** [comment]
How about `sudo nano /etc/openvpn/easy-rsa/pki/Default.txt`?

**Comment by 4s3ti** [comment]
closing this one as it doesn't seem pivpn-ish at all and no answer given back. 
and obviously those files need sudo to be edited as they are not owned by the user that is typically trying to edit it and they are not 777.
